packages/preview/g-exam/0.2.0/typst.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "g-exam"
+version = "0.2.0"
+compiler = "0.11.0"
+entrypoint = "g-exam.typ"
+authors = ["Andrés Giménez Muñoz"]
+license = "MIT"
+description = "Create exams with student information, grade chart, score control, questions, and sub-questions."
+keywords = ["exam", "test", "school", "question", "subquestion", "solutions", "teaching"]
+repository = "https://github.com/MatheSchool/typst-g-exam"
+categories = ["visualization", "paper", "office"]
```

Comment:
I didn't notice this the first time around, but I think only `office` makes sense from these. `visualization` is for plotting and stuff, while `paper` is for academic papers submitted to conferences.

Reply:
The template is oriented to the academic world. If you want to separate it from the research, you could create a new category, which has a lot of potential users. By exmaple ```school``` or ```academic``` can refer to academic world.

Reply:
That's what `disciplines = ["education"]` is for, which the template already contains.